Bridges/HttpKernel.php

```diff
@@ -100,6 +100,10 @@ public function onRequest(ReactRequest $request, HttpResponse $response)
         if ($this->application instanceof TerminableInterface) {
             $this->application->terminate($syRequest, $syResponse);
         }
+        
+        if (is_a($this->application, '\Illuminate\Contracts\Http\Kernel')) {
+            $this->application->terminate($syRequest, $syResponse);
+        }
```

Comment:
The only thing that Kernel implements, is just [mentioned contract](https://github.com/laravel/framework/blob/5.2/src/Illuminate/Contracts/Http/Kernel.php), that's why this one more case is required.